shared/typescript/ligo.ts

```diff
@@ -55,7 +55,7 @@ async function compileContract(
   main: string,
   dstFilePath: string
 ): Promise<void> {
-  const cmd = `ligo compile-contract ${srcFilePath} ${main} --brief --output=${dstFilePath}`;
```

Comment:
OOC Why the option change?

Reply:
they removed `--brief` option. Now its brief by default